src/main/webapp/script.js

```diff
@@ -30,62 +30,100 @@ async function getYouTubeComments() {
+ * and high magnitude comments
+ * @param {Array<sentimentBucket>} sentimentBucketList
+ */
+function getBarChart(sentimentBucketList) {
+  const CommentSentimentTable = new google.visualization.DataTable();
+  CommentSentimentTable.addColumn('number', 'InclusiveStart');
+  CommentSentimentTable.addColumn('string', 'SentimentRange');
+  CommentSentimentTable.addColumn('number', 'CommentCount');
 
-      CommentSentimentTable.addRow([rangeAsString, 
-          currentSentimentBucket.frequency, 
-          toTooltipString(highestMagnitudeComments)]);
-    }
+  // The json keys (ranges of scores) are sorted through their starting values
+  sentimentBucketList.forEach((sentimentBucket) => {
+    const inclusiveStart = sentimentBucket.intervalRange.inclusiveStart;
+    const exclusiveEnd = sentimentBucket.intervalRange.exclusiveEnd;
+    CommentSentimentTable.addRow(
+        [inclusiveStart, inclusiveStart + ' to ' + exclusiveEnd,
+          sentimentBucket.frequency]);
+  });
 
-    const options = {
-      'title': 'Comment Sentiment Range',
-      'width': CHART_WIDTH,
-      'height': CHART_HEIGHT,
-      'bar': {groupWidth: '100'},
-      'tooltip': {isHtml: true}
-    };
+  const options = {
+    'title': 'Comment Sentiment Range',
+    'width': CHART_WIDTH,
+    'height': CHART_HEIGHT,
+    'bar': {groupWidth: '100'},
+    'tooltip': {isHtml: true},
+  };
 
-    // Hide loading image once chart is drawn
-    document.getElementById('loading-img').style.display = 'none';
+  // Hide loading image once chart is drawn
+  document.getElementById('loading-img').style.display = 'none';
 
-    const view = new google.visualization.DataView(CommentSentimentTable);
-    const chart = new google.visualization.ColumnChart(
-        document.getElementById('chart-container'));
-    chart.draw(view, options);
+  CommentSentimentTable.sort({column: 0, desc: false});
+  const view = new google.visualization.DataView(CommentSentimentTable);
+  view.setColumns([1, 2]);
 
-    const averageContainer = document.getElementById('average-score-container');
-    averageContainer.innerHTML = 'Average Sentiment Score: ' + averageScore;
-  });
+  const chart = new google.visualization.ColumnChart(
+      document.getElementById('chart-container'));
+  chart.draw(view, options);
 }
 
+/**
+ * Create a word cloud based on the number of appearance for each word
+ * @param {Map<String:Integer>} wordFrequencyMap Map that contains
+ *                                top popular words and its appearance
+ */
+function getWordCloudChart(wordFrequencyMap) {
+  const data = [];
+  Object.keys(wordFrequencyMap).forEach((wordKey) =>
+    data.push({'x': wordKey, 'value': wordFrequencyMap[wordKey]}));
+  // Create a tag cloud chart
+  const chart = anychart.tagCloud(data);
+
+  chart.title('Most Popular Comments');
+  // Set array of angles to 0, make all the words display horizontally
+  chart.angles([0]);
+  chart.container('word-cloud');
```

Comment:
Should we name this word-cloud-container to match the naming convention of the other containers?